modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,139 @@
+= Portal
+
+`Unit ability`
+footnote:[
+include::special-rules:partial$no-such-ability-in-e40k.adoc[]
+]
+
+Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
+This can be an effective method to deploy reserve forces at the optimum time and place.
+
+Well-known examples include Necron portals and Eldar Webway portals.
+
+---
+
+== Data sheet variations of {portal}
+
+Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, as attributes in brackets.
+For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.
+
+Mostly, these attributes indicate what other friendly units can actually pass through the portal -- that is, which kinds of unit that portal permits you to embark/disembark during the game.
+
+* `_(INF)_`: Permits Infantry units.
+* `_(4+)_`: Permits Vehicle units that have an Armour value of `4+` or lower.
+* `_({walker})_`: Permits Vehicle units that have the {walker} ability, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that you intend to use with it, through the `_(Roll)_` or `_(Auto)_` attributes.
+See <<Portal Reserves>> for more about this.
+
+TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
+They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list must all be of one of these types, never a mix of both.
```

Comment:
As these will be mutually exclusive at the army list level rather than the player's choice level, I think it would be safer (and less confusing) to say 'will all be' rather than 'must all be'. So:

```suggestion
TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list will all be of one of these types, never a mix of both.
```

Reply:
Hmmm. Let's break this down a bit:

- 'must' works for all time frames, but does make more sense from the point of view of the designer than the player, I agree
- 'will' implies a **future** where the army lists will exist but might not yet — it's still rather beholden to the designer's point of view
- 'are' would be more clear that we create the lists with this limitation in mind, and so "that's how you, the player, will find them", and is clearly pertinent for both existing and future lists

So, perhaps something like this:

```suggestion
TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
They are mutually exclusive, for both thematic and practical reasons -- so all {portal} units in a given army list are of one type, never a mix of both.
```

Reply:
👍